src/main/java/com/example/demo/Models/Actor.java

```diff
@@ -4,18 +4,33 @@
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.JoinColumn;
 
 @Data
 @Entity
 public class Actor {
-    private @Id @GeneratedValue Long id;
+    private @Id @GeneratedValue(strategy = GenerationType.IDENTITY) Long id;
     private String name;
 
+    @JsonIgnore
+    @ManyToOne
```

Comment:
This type of relationship means that an **_Actor_** is related with a unique **_Movie_**, is that ok? 
I think an **_Actor_** will be part of many **_Movies_**